scripts/pcm-dpc/covid-19/dpc-covid19-ita_StatisticalVariable.mcf

```diff
@@ -0,0 +1,95 @@
+Node: dcid:count_MedicalConditionIncident_COVID_19_HospitalizedsWithSymptoms
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+medicalStatus: dcs:HospitalizedsWithSymptoms
+
+Node: dcid:count_MedicalConditionIncident_COVID_19_IntensiveCare
```

Comment:
Can we reuse https://github.com/datacommonsorg/data/blob/master/scripts/covid_tracking_project/historic_state_data/COVIDTracking_States_StatisticalVariables.mcf#L65